client/lib/stats/stats-list/test/index.js

```diff
@@ -0,0 +1,185 @@
+/**
+ * External Dependencies
+ */
+import { assert } from 'chai';
+import sinon from 'sinon';
+
+/**
+ * Internal Dependencies
+ */
+import useMockery from 'test/helpers/use-mockery';
+import useFakeDom from 'test/helpers/use-fake-dom';
+
+describe( 'StatList', () => {
+	let StatList, statsParser, data
+
+	useMockery( mockery => {
+		mockery.registerMock( 'lib/wp', require( './mocks/lib/wp' ) );
+		mockery.registerMock( 'lib/user', require( './mocks/lib/user' ) );
+	} );
+
+	useFakeDom();
```

Comment:
So leave it as it is :) I was referring to the cases when it's really not necessary.

We have still lots of tests to move, this should be our focus now. We have much bigger issues in other places :D